code/src/App.js

```diff
@@ -1,9 +1,165 @@
+import { useState } from 'react';
+
+import DropdownLocation from 'DropdownLocation'
+import NameInput from 'NameInput'
+import RadioBtnAge from 'RadioBtnAge'
 import React from 'react'
+import Summary from 'Summary'
+import Header from './Header'
+
+import './index.css';
```

Comment:
Clean your imports up :) 
1. internal imports from react
2. components from node_modules
3. your components
4. CSS and another imports